edb/ir/staeval.py

```diff
@@ -374,8 +376,12 @@ def object_type_to_python_type(
         else:
             pytype = scalar_type_to_python_type(ptype, schema)
 
-        ptr_card = p.get_cardinality(schema)
-        is_multi = ptr_card.is_multi()
+        ptr_card: qltypes.SchemaCardinality = p.get_cardinality(schema)
+        try:
```

Comment:
Probably just do a `not is_known()` check instead?

Reply:
Yeah, that's way clearer. Duh.